src/main/java/Doubler.java

```diff
@@ -1,6 +1,6 @@
 public class Doubler {
 
     public int doubleIt(int val) {
```

Comment:
Are you sure you did not want to multiply by 8?

Reply:
No, it said to double it!

Reply:
I am able to comment here